src/main/java/org/sciborgs1155/robot/Intake/RealIntake.java

```diff
@@ -0,0 +1,22 @@
+package org.sciborgs1155.robot.Intake;
+
+import static org.sciborgs1155.robot.Intake.IntakeConstants.*;
+
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+
+public class RealIntake implements IntakeIO {
+  CANSparkMax motor = new CANSparkMax(7, MotorType.kBrushless);
+  RelativeEncoder encoder = motor.getEncoder();
+
+  @Override
+  public double getSpeed() {
+    return encoder.getVelocity() * MOTOR_RADIUS;
```

Comment:
same thing as with hopper. Also, note that you'll actually be addressing stuff like this in setting the conversion factors for the encoders